src/plugins/requestAuthPlugin.js

```diff
@@ -107,17 +107,17 @@ export const applyRequestAuthorizer$ = requestAuthorizer$ => request => {
  * Get an anonymous code from the API that can be used to generate an oauth
  * access token
  *
- * @param {Object} config { OAUTH_AUTH_URL, oauth }
+ * @param {Object} server the Hapi app server instance
```

Comment:
Ohhhh! I'm just seeing this comment ---- you can ignore my questions from the comments below :) 

I'll update that last reference to use server.settings.app and presume to use server.settings.app everywhere then?